tests/generators/test_huggingface.py

```diff
@@ -121,6 +133,18 @@ def test_model(hf_generator_config):
         assert item is None  # gpt2 is known raise exception returning `None`
 
 
+def test_model_chat(mocker, hf_generator_config):
+    # uses a ~350M model with chat support
+    g = garak.generators.huggingface.Model(
+        "microsoft/DialoGPT-small", config_root=hf_generator_config
+    )
+    mock_format = mocker.patch.object(
+        g, "_format_chat_prompt", wraps=g._format_chat_prompt
+    )
+    g.generate("Hello world!")
```

Comment:
Can we check if the output is coming in the desired format? This would then validate the code better